Library/Homebrew/dev-cmd/pr-pull.rb

```diff
@@ -104,7 +104,7 @@ def signoff!(path, pr: nil, dry_run: false)
     if pr
       # This is a tap pull request and approving reviewers should also sign-off.
       tap = Tap.from_path(path)
-      trailers += GitHub.approved_reviews(tap.user, "homebrew-#{tap.repo}", pr).map do |r|
+      trailers += GitHub.approved_reviews(tap.user, tap.full_name.split("/").last, pr).map do |r|
```

Comment:
A `Tap#full_repo` or something to provide this might be worth considering if we use this in multiple places.